homeassistant/components/webostv/trigger.py

```diff
@@ -0,0 +1,60 @@
+"""webOS Smart TV trigger dispatcher."""
+from __future__ import annotations
+
+from types import ModuleType
+from typing import cast
+
+from homeassistant.components.automation import (
+    AutomationActionType,
+    AutomationTriggerInfo,
+)
+from homeassistant.const import CONF_PLATFORM
+from homeassistant.core import CALLBACK_TYPE, HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .triggers import turn_on
+
+TRIGGERS = {
+    "turn_on": turn_on,
+}
+
+
+def _get_trigger_platform(config: ConfigType) -> ModuleType:
```

Comment:
We can define a protocol for the trigger modules so we don't need to use hasattr and getattr below.

Example:
https://github.com/home-assistant/core/blob/927bfb6d1ad11472d5a3dd7d76774e9543754cff/homeassistant/helpers/service_platform.py#L45-L54